src/main/java/Homework3/Task2_2.java

```diff
@@ -0,0 +1,53 @@
+package Homework3;
+
+import java.util.Random;
+
+public class Task2_2 {
+    public static void main(String[] args) {
+// Create and print array
+        System.out.println("Array: ");
+        int[] arr = new int[10];
+        Random rand = new Random();
+        for (int i = 0; i < 10; i++) {
+            arr[i] = rand.nextInt(100);
+            System.out.print(arr[i] + "  ");
+        }
+        System.out.println("\n\nProcess: ");
+        // method sort
+        sort(arr, arr.length);
+        // print sorted array
+        System.out.println("\nSorted array: ");
+        for (int i = 0; i < arr.length; i++) {
+            System.out.print(arr[i] + "  ");
+        }
+
+
+    }
+// method sort
+    static void sort(int[] arr, int len) {
```

Comment:
good idea, but I think this argument int[] arr is enough 
you can easily get the length from arr if you needed